back/lib/callable/class_methods.rb

```diff
@@ -6,17 +6,16 @@ module Callable
   module ClassMethods
     def call(*args)
       begin
-        @result = Callable::Result.new
-        new(*args).send(_call_alias)
+        callable_instance = new(*args)
```

Comment:
A bit convoluted, but if it works that's most important. I clearly prefer the python way here 😜

Reply:
Actually why not using something like:
```ruby
def result
  @result ||= Callable::Result.new
end
```
to initialize the instance var instead of using `instance_variable_set`.